### ISSUE TITLE
Bug - Console spamming

### ISSUE THREAD

**Issue by dsatgh** [legoboyvdlp/A320-family#297]
These three lines are spamming my console every 0.1 seconds:

```
3770.35 [ALRT]:nasal      Nasal runtime error: floating point error in math.cos()
3770.35 [ALRT]:nasal        at /home/daniel/myFGFS/flightgear/share/fgdata/Nasal/geo.nas, line 172
3770.35 [ALRT]:nasal        called from: /home/daniel/.fgfs/Aircraft/org.flightgear.fgaddon.stable_2020/Aircraft/A320-family/Nasal/FMGC/flightplan.nas, line 785
3770.35 [ALRT]:nasal        called from: /home/daniel/.fgfs/Aircraft/org.flightgear.fgaddon.stable_2020/Aircraft/A320-family/Nasal/FMGC/flightplan.nas, line 945
```

Steps to reproduce the behaviour:
1. Start the aircraft
2. Make a flightplan
3. Make the flight
4. Park the aircraft
5. Rudely turn off the engines
6. After a few seconds see error

 - OS: Debian 12
 - FlightGear version: 2020.3.18
 - A320 Version: Revision 52

**Comment by legoboyvdlp** [comment]
Possible division by zero, possible fix by fa6ee1d5, please confirm

**Comment by dsatgh** [comment]
Confirmed. It's fixed. Thanks a lot.
